addons/knowledge/static/src/js/knowledge_controller.js

```diff
@@ -247,11 +247,15 @@ const KnowledgeFormController = FormController.extend({
         const result = await this._toggleFavourite(id);
         $(event.target).toggleClass('fa-star-o', !result).toggleClass('fa-star', result);
         this._rpc({
-            route: '/knowledge/get_favourite_tree'
+            route: '/knowledge/get_favourite_tree',
+            params: {
+                res_id: id,
+            }
         }).then(favouriteTemplate => {
             self.$(".o_favourite_container").replaceWith(favouriteTemplate);
             this.renderer._setTreeListener();
```

Comment:
this line is not useful, this was in case the _setTreeListener would handle the favourite part too, which is not the case